packages/eslint-plugin-query/src/rules/prefer-query-object-syntax/prefer-query-object-syntax.test.ts

```diff
@@ -1,7 +1,11 @@
 import { rule, name } from './prefer-query-object-syntax'
-import { createRuleTester, normalizeIndent } from '../../utils/test-utils'
+import { normalizeIndent } from '../../utils/test-utils'
+import { ESLintUtils } from '@typescript-eslint/utils'
 
-const ruleTester = createRuleTester()
```

Comment:
while `createRuleTester` is using `@typescript-eslint/parser`, it didn't understand type annotations. instead, it inferred them as `BinaryExpression` instead. Thus, I decided to drop it.